cmd/alertmanager/main.go

```diff
@@ -80,12 +80,19 @@ var (
 		},
 		[]string{"handler", "method"},
 	)
+	clusterEnabled = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "alertmanager_cluster_enabled",
+			Help: "Whether the clustering is enabled or not.",
```

Comment:
Can we add "Shows" or "Indicates" at the front of the help text?

Reply:
sure